src/localization/en.js

```diff
@@ -122,12 +122,11 @@ export default {
     },
 
     currentBenefits: {
-      currentBenefits_v1:  `Current Benefits`,
-      selectBenefits_v1:   `Select the benefits you currently receive`,
-      hasSection8Label_v1: `Do you have Section 8 Housing?`,
-      hasSection8Hint_v1:  `Section 8 provides rental housing assistance.`,
-      hasSnapLabel_v1:     `Do you have SNAP?`,
-      hasSnapHint_v1:      `SNAP provides assistance with buying food`,
+      currentBenefits_v1:      `Current Benefits`,
+      selectBenefits_v1:       `Select the benefits you currently receive`,
+      has_section8_label_v1:   `Do you have Section 8 Housing?`,
+      has_snap_label_v1:       `Do you have SNAP?`,
+      has_xrayVision_label_v1: `Do you have X-ray vision?`,
```

Comment:
lol! Maybe we should keep it in, though. I kind of like it. It'll be an easter egg for our fans :D

Reply:
And it'll show up on the localization report! That's hilarious!

Reply:
Haha, damn it, I added that to test arbitrary numbers of benefits. I thought I had removed it!